tests/test_data_types.py

```diff
@@ -586,6 +586,31 @@ def test_table_default():
     }
 
 
+def test_big_table_throws_error_that_can_be_overridden(live_mock_server, test_settings):
+    test_settings.update({"table_raise_on_max_row_limit_exceeded": True})
+    run = wandb.init(settings=test_settings)
+
+    # make this smaller just for this one test to make the runtime shorter
+    wandb.Table.MAX_ARTIFACT_ROWS = 10
```

Comment:
Could you please replace this and below with 
```
with mock.patch("wandb.Table.MAX_ARTIFACT_ROWS", 10):
```
We prefer avoiding relying on `finally` statements. Also, this will reset this attr globally for the pytest worker, so there may be unexpected future test failures (e.g. if a new test is added where this is assumed to have a different value).

Reply:
done